src/context/WalletProvider/NewWalletViews/routes/LedgerRoutes.tsx

```diff
@@ -124,6 +125,10 @@ export const LedgerRoutes = () => {
     await handlePair()
   }, [handleClearPortfolio, handlePair])
 
+  const handleConnectReadOnly = useCallback(() => {
```

Comment:
Easy-as, since we don't disconnect anymore on refresh with the read-only, "Connecting as read-only" is as simple as closing the modal